rust/gui-client/src-tauri/src/client.rs

```diff
@@ -66,14 +64,10 @@ pub(crate) fn run() -> Result<()> {
 
     match cli.command {
         None => {
-            match elevation::check() {
-                // We're already elevated, just run the GUI
+            match elevation::is_normal_user() {
+                // Our elevation is correct (not elevated), just run the GUI
                 Ok(true) => run_gui(cli),
-                Ok(false) => {
-                    // We're not elevated, ask Powershell to re-launch us, then exit. On Linux this is completely different.
-                    elevation::elevate()?;
-                    Ok(())
-                }
+                Ok(false) => bail!("The GUI should run as a normal user, not elevated"),
```

Comment:
It might work, but I'll wait and see if anyone runs into this. I'm not sure if there is a way to `sudo` run a GUI without having a terminal open, and the error shows up in the terminal, I just checked.